hw/chisel/src/main/scala/snax/csr_manager/CsrManagerWithReadOnly.scala

```diff
@@ -0,0 +1,63 @@
+package snax.csr_manager
+
+import snax.utils._
+
+import chisel3._
+import chisel3.util._
+
+class CsrManagerWithReadOnlyIO(
+    csrNum: Int,
+    csrNumReadOnly: Int,
+    csrAddrWidth: Int
+) extends CsrManagerIO(csrNum, csrAddrWidth) {
+
+  val read_only_csr = Input(Vec(csrNumReadOnly, UInt(csrAddrWidth.W)))
+
+}
+
+class CsrManagerWithReadOnly(
+    csrNum: Int,
```

Comment:
I like to re-name the `csrNum` to `csrNumReadWrite`. Reason: I remember when I was working before we had conventional signal names like `RW` - read and write, `RO` - read only, and `WO` write only. We even had the `pass_through` registers which are custom controlled by some use. What do you think? Would it be nice to re-write it as `csrNumReadWrite`?

Reply:
Changed!